tests/console/validate_autoyast.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: smoke test for autoyast post-installation
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use utils;
+use y2logsstep;
+
+sub run {
+    my $self     = shift;
+    my $username = "bernhard";
```

Comment:
just use `$testapi::username`

Reply:
and as you `use testapi;`, `$username` should already be set to `bernhard`.

Reply:
this is what I meant